src/api-reference/accounting-integration/v4.accountingintegration-schema.markdown

```diff
@@ -180,9 +180,11 @@ Please note that if no additional configuration is needed the list of configurat
 |Name|Type|Format|Description|
 |---|---|---|---|
 `erpId`|`string`|`UUID`|Universally unique identifier of the ERP.|
+`companyEdition`|`array`|[`CompanyEdition`](#companyedition)|Company edition in Concur (Standard or Professional).|
```

Comment:
can you change to: "The company edition offering in SAP Concur - STANDARD for Standard Edition, or PROFESSIONAL for Professional Edition."